201901_gaussian_fg_lensed_cmb_realistic_noise/simulation.cfg

```diff
@@ -0,0 +1,77 @@
+
+output_nside = 4096
+output_folder = output/$output_nside
+channels = all
+unit = uK_CMB
```

Comment:
All output maps should be in `uK_CMB`, thumbs up reaction if ok, otherwise please answer to this thread.